instrumentation/sidekiq/lib/opentelemetry/instrumentation/sidekiq/patches/launcher.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module Sidekiq
+      module Patches
+        module Launcher
+          def ❤
```

Comment:
The method signature changes in earlier versions of `sidekiq 4.x.x`, I bumped the appraisal to be the latest `4.2.x` release. 

https://github.com/mperham/sidekiq/blob/v4.1.3/lib/sidekiq/launcher.rb#L74